packages/server/src/integrations/tests/sql.spec.ts

```diff
@@ -551,42 +190,6 @@ describe("SQL query builder", () => {
     })
   })
 
-  it("should handle table names with dashes when performing a LIKE in MySQL", () => {
-    const tableName = "Table-Name-With-Dashes"
```

Comment:
Is there a test case for this as well already?

Reply:
Yes, we use UUIDs a lot for table names elsewhere.